tools/ant_world_db_creator/ant_world_db_creator.cc

```diff
@@ -229,9 +229,7 @@ main(int argc, char **argv)
     if (argc > 1) {
         // Create route object and load route file specified by command line
         AntWorld::RouteContinuous route(0.2f, 800);
-        if (!route.load(argv[1])) {
-            return 1;
-        }
+        route.load(argv[1]);
```

Comment:
Could you fix ardin_mb as well? It tests the return value of this at https://github.com/BrainsOnBoard/bob_robotics/blob/master/projects/ardin_mb/state_handler.cc#L33